edx/edx-enterprise
# ENT-519: Create Audit enrollment in ecommerce system when user enrolls in the audit mode in enterprise landing page.

FYI -- @douglashall , @brittneyexline , @zubair-arbi , @asadiqbal08 

**Description:** 
Currently, when users come to course about pages in edx.org and click enroll, a call is sent to the ecommerce api that writes both the enrollment record and places an audit order in the ecommerce system. The same logic does not currently exist in the landing page flow, so we need to make sure an audit order is placed when users come to the enterprise landing page.

**JIRA:** [ENT-519](https://openedx.atlassian.net/browse/ENT-519)

**Installation instructions:** Simply update edx-enterprise version to this branch.

**Testing instructions:**

1. Create a course with an "audit" mode.
2. Go to enterprise landing page for some enterprise customer and the course you created above.
3. Click on the enroll button.
4. Learner should be enrolled in the course.
4. Open ecommerce admin page and go to "Orders" list page at "/admin/order/order/". There should be a completed order for the above enrollment.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
